assets/pms.json

```diff
@@ -697,6 +697,20 @@
     "aa_fn": "pm25.fFLYING_03",
     "family": "Pikachu_23fly"
   },
+  {
+    "dex": 25,
+    "isotope": "_cFEB_2019",
```

Comment:
I recommend to use shorter string for isotope.
That will affect url length. (browser limitation)